ui/src/ImageUpload.tsx

```diff
@@ -1,23 +1,33 @@
 import * as React from 'react';
+import axios from 'axios';
 
-class ImageUpload extends React.Component {
+class ImageUpload extends React.Component<any, any> {
```

Comment:
Is adding the `<any, any>` for Typescript?

Reply:
yes I added this for the typescript

Reply:
@kosmond13 For reference, this is telling Typescript the data types to use for the Props and State, respectively.

@jphilly11 We should refactor the state to be using a Typescript type.  I'll provide an example.